doc/changelogs/CHANGELOG_V10.md

```diff
@@ -58,7 +58,7 @@
 * EventEmitter
   * The `EventEmitter.prototype.off()` method has been added as an alias for `EventEmitter.prototype.removeListener()`. [[`3bb6f07d52`](https://github.com/nodejs/node/commit/3bb6f07d52)]
 * File System
-  * The `fs.promises` API provides experimental promisified versions of the `fs` functions. [[`329fc78e49`](https://github.com/nodejs/node/commit/329fc78e49)]
+  * The `fs/promises` API provides experimental promisified versions of the `fs` functions. [[`329fc78e49`](https://github.com/nodejs/node/commit/329fc78e49)]
```

Comment:
This change is actually wrong. Originally it did indeed land as `fs.promises`. I would have kept it as is therefore. There should be another commit that changed it later on.

Reply:
A good alternative would be to add the commit that changed it to `fs/promises`.

Reply:
Feel free to do either, I simply relied on @devsnek's and others reviews.

Reply:
But this isn't a commit message. This is a textual list of notable changes. Unless v10.0.0 shipped with fs.promises rather than fs/promises, this change is correct, IMO. Later on, when the commits are all listed, it says `fs.promises` and that is correct for the reason you identify here. But this section should not be true to specific commits but instead explain the notable change in the release.